src/main/java/org/arquillian/container/chameleon/ChameleonConfiguration.java

```diff
@@ -141,4 +141,18 @@ private String getOutputDirectory() {
             }
         }
     }
+
+    public static boolean isSupported(String source) throws Exception {
```

Comment:
I would remove `static` qualifier but that's all, for me it is not bad at all having it here.